frontend/src/components/elements/BetaTable/BetaTable.tsx

```diff
@@ -0,0 +1,141 @@
+/**
+ * @license
+ * Copyright 2018-2020 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { range } from "lodash"
+import React, { ReactElement } from "react"
+
+import { Arrow as ArrowProto } from "autogen/proto"
+import withFullScreenWrapper from "hocs/withFullScreenWrapper"
+import { Arrow } from "lib/Arrow"
+import {
+  StyledEmptyTableCell,
+  StyledTable,
+  StyledTableCell,
+  StyledTableCellHeader,
+  StyledTableContainer,
+} from "./styled-components"
+
+export interface TableProps {
+  element: ArrowProto
+}
+
+export function BetaTable(props: TableProps): ReactElement {
+  const { data, styler } = props.element
+
+  const table = new Arrow(data, styler || undefined)
+  const { tableId, tableStyles, caption } = table
+
+  const { headerRows, rows, columns } = table.dimensions
+  const allRows = range(rows)
+  const columnHeaders = allRows.slice(0, headerRows)
+  const dataRows = allRows.slice(headerRows)
+
+  return (
+    <StyledTableContainer data-testid="stTable">
+      {tableStyles && <style>{tableStyles}</style>}
```

Comment:
Is there anyway to convert this to a styled component? I'm guessing this is a string of CSS?